02 01 CounterButtonComponent

ase444ka opened the pull request:
-completed

ase444ka committed:
02 01 CounterButtonComponent
-completed

ase444ka committed:
Merge branch 'master' of github.com:js-tasks-ru/vue-20210119_istomina-asia into master

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

```diff
MeetupPageComponent
- MeetupPage должен быть определён
- MeetupView должен иметь обязательный параметр meetup с объектом митапа
- MeetupView должен показывать изображение, описание, информацию и программу митапа соответствующими компонентами
- MeetupCover должен иметь необязательные строковые параметры link и title
- MeetupCover должен выводить изображение митапа
- MeetupCover не должен удалять изображение по умолчанию в случае отсутствия ссылки на изображение
- MeetupCover должен выводить название митапа
- MeetupDescription должен иметь необязательный строковый параметр description
- MeetupDescription должен выводить описание митапа
- MeetupInfo должен иметь обязательные строковые параметры organizer, place и обязательный параметр с датой date
- MeetupInfo должен выводить краткое описание митапа
- MeetupAgenda должен иметь параметр массив agenda
- MeetupAgenda должен выводить программу через компоненты MeetupAgendaItem
- MeetupAgendaItem должен иметь обязательный параметр объект agendaItem
- MeetupAgendaItem должен выводить время
- MeetupAgendaItem должен выводить заголовок по умолчанию
- MeetupAgendaItem должен выводить правильную иконку для регистрации
- MeetupAgendaItem должен выводить доклад
```
```diff
Компонент CounterButton
+ CounterButton должен иметь числовой входной параметр count со значением 0 по умолчанию
+ CounterButton должен рендерить кнопку с текстом count
+ CounterButton должен иметь count 0 по умолчанию
+ CounterButton должен порождать событие increment с увеличенным на 1 значением по клику
```

ase444ka committed:
02 02 MeetupPageComponent in progress
-added components:
    MeetupPage
    MeetupAgenda
    MeetupCover
    MeetupDescription
    MeetupView

ase444ka committed:
02 02 MeetupPageComponent in progress
-fixed MeetupCover's params

ase444ka committed:
02 02 MeetupPageComponent in progress
-fixed MeetupAgenda's params

ase444ka committed:
02 02 MeetupPageComponent in progress
 -added Meetupinfo component

ase444ka committed:
02 02 MeetupPageComponent completed

ase444ka committed:
Merge branch 'master' of github.com:js-tasks-ru/vue-20210119_istomina-asia into master

ase444ka committed:
02 03 MeetupsCalendar in progress
-made switching months by pressing rangepicker__selector-control

ase444ka committed:
02 03 MeetupsCalendar in progress
-made rendering correct calendar cells

ase444ka committed:
02 03 MeetupsCalendar in progress
-made computeShedule method

ase444ka committed:
02 03 MeetupsCalendar in progress
-made meetup's titles rendering in calendar

ase444ka committed:
02 03 MeetupsCalendar in progress
-made minimum refactor

ase444ka committed:
02 03 MeetupsCalendar in progress
-shedule made to be computed property instead of option

ase444ka committed:
02 03 MeetupsCalendar completed
